doc/md/fundamentals/1-basic-syntax/5-characters-strings.md

```diff
@@ -18,41 +18,63 @@ let symbol : Char = '✮';
 
 You can convert a `Text` value `t` to a `Char` iterator using either `t.chars()` or, equivalently, `Text.toIter(t)`.
 
-```motoko no-repl
+:::info [Iter](https://internetcomputer.org/docs/motoko/base/Iter)
+An `Iter<T>` is an object that sequentially produces values of specified type `T` until no more values remain.
+:::
+
```

Comment:
I actually think this example is way too complicated for this location (the code is also more complicated than it needs to be):

How about we just implement `reverse` and call it (without even wrapping in actor, but you can if you want).

````suggestion
``` motoko
import Char "mo:base/Char";

func reverse(t: Text) : Text {
  var result = "";
  for (c in t.chars()) {
    result := Char.toText(c) # result
  };
  result;
};

reverse("Motoko");
```
The operator `#` concatenates two `Text` values.

```
````